src/server/game/Entities/Unit/StatSystem.cpp

```diff
@@ -562,6 +566,17 @@ void Player::UpdateMastery()
     }
 }
 
+void Player::UpdateVersatility()
+{
+    // No proof that CR_VERSATILITY_DAMAGE_DONE is allways = PLAYER_VERSATILITY
+    SetUInt32Value(PLAYER_VERSATILITY, GetUInt32Value(PLAYER_FIELD_COMBAT_RATING_1 + CR_VERSATILITY_DAMAGE_DONE));
```

Comment:
field seems unused by the client anyway, always calculating it from PLAYER_FIELD_COMBAT_RATING_1

Reply:
So it's ok to have it like this.

Reply:
I mean, what is the value that sniffs show? a float percentage or rating?

Reply:
Just Seen Same value as in cr Field... But unsure

Reply:
Never seen difference between 
```
PLAYER_FIELD_COMBAT_RATING_1 + CR_VERSATILITY_DAMAGE_DONE
PLAYER_FIELD_COMBAT_RATING_1 + CR_VERSATILITY_DAMAGE_TAKEN
PLAYER_FIELD_COMBAT_RATING_1 + CR_VERSATILITY_HEALING_DONE
```